hack/update-proto.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
 API_ROOT="${ROOT}/${API_PATH-"pkg/api/v1"}"
 
-go get k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
+go install k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo@latest
```

Comment:
Does this file even work?

Reply:
yup, I used it to generate the new CRI grpc files